modules/webgl/src/classes/texture.js

```diff
@@ -195,14 +195,23 @@ export default class Texture extends Resource {
         type: this.type,
         dataFormat: this.dataFormat,
         border: this.border,
-        mipmaps: false
+        mipmaps: this.mipmaps
```

Comment:
This logic doesn't seem correct to me. What if you resize from a POT to NPOT size?

A better solution to me would be to just pass in mipmapping as an additional parameter (default to false).

Reply:
@tsherif 

inside `resize`, it calls `initialize`, which will check if the texture is `NPOT` or not and disable mipmap for `NPOT`. But it is a problem if the user resizes the texture from `POT` to `NPOT` and then back to `POT`, the `mipmap` will be false.